docs/src/assets/scripts.js

```diff
@@ -29,45 +29,66 @@ const maxColWidth = '280px';
 let winWidth = getWinWidthStr();
 
 const establishDisplay = () => {
-  const { bookPath, hasOtp } = currentPage;
```

Comment:
You merged this file incorrectly and blew away some important changes

Reply:
When I kept those changes the bookPath in the left path no longer hid. 
I saved the revisions that weren't merged.

Reply:
You'll have to have Hamir fix it then, because the changes are important

Reply:
This file is still wrong. It doesn't have the changes he's made lately... it has dropped the abstraction of displayFor and it has dropped the Github Discussions search entries